docs/install-azure-repos.md

```diff
@@ -0,0 +1,230 @@
+[Go back to the main documentation page](../README.md)
+
+# Installing Frogbot on Azure Repos repositories
+
+To install Frogbot on Azure Repos repositories:
```

Comment:
To install Frogbot on Azure Repos repositories:
-->
To install Frogbot on Azure Repos repositories, follow these steps.

Reply:
done